spec/javascripts/health-care/components/Nav.spec.jsx

```diff
@@ -17,12 +45,20 @@ describe('<Nav>', () => {
       expect(activeLinks[0].getAttribute('href')).to.equal(path);
     };
 
+    const expectTwoActiveLinks = (component, path) => {
```

Comment:
Rename this to reflect the semantics more?

We are expecting nav and subnav both to be correctly active, not just that 2 links are active.